desk/app/activity.hoon

```diff
@@ -857,8 +864,9 @@
   =/  volume  (get-volume:evt volume-settings -.event)
   ::TODO  support other event types
   =*  is-msg  ?=(?(%dm-post %dm-reply %post %reply) -<.event)
-  =*  supported
-    |(is-msg ?=(?(%dm-invite %chan-init) -<.event))
+  =*  inits  ?=(?(%dm-invite %chan-init) -<.event)
+  =*  flags  ?=(?(%flag-post %flag-reply) -<.event)
```

Comment:
Consider `is-init` and `is-flag` for consistency and clarity. (Arguably, `is-supported` too, but that one was already there.)